railties/lib/rails/generators/rails/app/templates/config/environments/production.rb.tt

```diff
@@ -20,16 +22,12 @@ Rails.application.configure do
   # key such as config/credentials/production.key. This key is used to decrypt credentials (and other encrypted files).
   # config.require_master_key = true
 
-  # Disable serving static files from `public/`, relying on NGINX/Apache to do so instead.
-  # config.public_file_server.enabled = false
+  # Cache assets for far-future expiry since they are all digest stamped.
+  config.public_file_server.headers = { "Cache-Control" => "public, max-age=#{1.year.to_i}" }
```

Comment:
```suggestion
  config.public_file_server.headers = { "cache-control" => "public, max-age=#{1.year.to_i}" }
```

Since new applications get Rack 3